bundles/CoreBundle/Resources/translations/en.json

```diff
@@ -944,6 +944,8 @@
   "redirect_performance_warning": "Using this feature has very bad effects on the performance of the entire application, do you really want to continue?",
   "Uncategorized": "Uncategorized",
   "methods": "Methods",
+  "export_configuration": "Export As JSON",
```

Comment:
better Export Configuration as JSON
same for import

Reply:
Done